### DISCUSSION TITLE
Question Results AutoML Benchmark

### DISCUSSION THREAD

**Discussion by cynthiamaia** [openml/automlbenchmark#485] [Q&A]
I would like to know where are the results of the best pipelines found by the frameworks, is it in the log?

**Comment by PGijsbers** [comment]
It depends on the framework configuration whether or not they are saved and how. We don't have a standardized way of recording the different pipelines, but generally there will be a log or leaderboard file. 
For example, `GAMA`'s evaluated pipelines (with their internal validation score) can be found in its `logs.zip` (specifically `logs/TASKNAME/FOLD/evaluations.log`) and the process of ensemble construction in `logs/TASKNAME/FOLD/gama.log`.
For H2O it's stored in a model directory, e.g., `https://test.openml.org/amlb/classification_1h8c/h2oautoml.openml_s_271.1h8c_gp3.aws.20211118T222538/aws.openml_s_271.1h8c_gp3.ada.2.h2oautoml/output/models/ada/2/`.
